pkg/cmd/attestation/verify/verify_integration_test.go

```diff
@@ -76,15 +76,6 @@ func TestVerifyIntegration(t *testing.T) {
 		require.ErrorContains(t, err, "expected SourceRepositoryOwnerURI to be https://github.com/fakeowner, got https://github.com/sigstore")
 	})
 
-	t.Run("with invalid owner and invalid repo", func(t *testing.T) {
```

Comment:
The `with repo which not matches SourceRepositoryURI` and `with owner which not matches SourceRepositoryOwnerURI` test cases cover the possible scenarios.